### DISCUSSION TITLE
关于pnpm 依赖提升的问题

### DISCUSSION THREAD

**Discussion by windole** [youzan/vant#11322] [Q&A]
大佬好，有个pnpm的问题想请教下。jest相关依赖都在vant-cli中，在根工作区和vant包中都未显式依赖，在vant包中相当于幽灵依赖，为何vant中单元测试文件中还是可以使用jest。
安装依赖后查看node_modules，jest的相关依赖都已提升到root,但项目中没有找到hoist的相关配置，还望大佬指教。

**Comment by chenjiahan** [comment]
vant 包没有直接引用 jest 吧，都是间接引用，不存在幽灵依赖
